app/models/tslr_claim.rb

```diff
@@ -31,7 +31,7 @@ class TslrClaim < ApplicationRecord
   validates :claim_school,                      on: [:"claim-school", :submit], presence: {message: "Select a school from the list"}
   validates :current_school,                    on: [:"current-school", :submit], presence: {message: "Select a school from the list"}
 
-  validates :qts_award_year,                    on: [:"qts-year", :submit], inclusion: {in: VALID_QTS_YEARS, message: "Select the academic year you were awarded qualified teacher status"}
+  validates :qts_award_year,                    on: [:"qts-year", :submit], inclusion: {in: ClaimsController.helpers.options_for_qts_award_year.map(&:last), message: "Select the academic year you were awarded qualified teacher status"}
```

Comment:
Is there a way to do this without calling controller code from the model?

Reply:
I've added a commit that extracts this code out to a shared `QtsYears` model. I'm not sure if it's a conventional thing to do in Rails, I'm open to suggestions

Reply:
I also don't know if it's conventional, but I like it 🤷‍♂.

Reply:
The pattern we've followed elsewhere has been to use an `enum` column, e.g. `employment_status` and the [forthcoming student loan plan questions](https://github.com/DFE-Digital/dfe-teachers-payment-service/compare/335-capture-student-loan-repayment-plan). If we wanted to keep things really simple, we might look to turn this column into an enum and then hard-code the form fields based on the available options. The actual options aren't going to change so there is less of a need for the options, etc to be dynamically generated using code.

Reply:
I've changed the field to be an enum. The migration may not work and will probably need a `db:reset` for it to work properly, not sure how we're going to do that on staging (maybe just manually destroy the database and start fresh)?